tests/AssetManagerTest/ModuleTest.php

```diff
@@ -150,6 +150,10 @@ public function testOnBootstrap()
     {
         $applicationEventManager = new EventManager();
 
+        if (!method_exists($applicationEventManager, 'getListeners')) {
```

Comment:
For this, check out https://github.com/zendframework/zend-eventmanager/blob/510213c7c31268ca8c7c2fe4c4ba63dc5581246b/src/Test/EventListenerIntrospectionTrait.php - it has methods for asserting listeners were attached at a specific priority, methods for pulling listeners, etc.